### ISSUE TITLE
Login error on linux (Debian 11)

### ISSUE THREAD

**Issue by AndyFicile** [SineVector241/VoiceCraft-MCBE_Proximity_Chat#115]
Hi, can you help me? When the server is host on linux I get this error when I try to login:
[Error] [VoiceCraftServer]: Server Failed - Reason: Cannot access a disposed object.
Object name: 'System.Net.HttpListenerResponse'., Exception Type: ObjectDisposedException
[Error] [VoiceCraftServer]: Shutting down server in 10 seconds...

I don't have this problem on windows

**Comment by SineVector241** [comment]
Huh. That's interesting. I don't know why that is happening.

**Comment by SineVector241** [comment]
The server works on ubuntu but I never tested on debian so it probably doesn't work that well on it. This all might be fixed in the next update, but at the moment, I don't know why that is happening.

**Comment by NeoNathoon** [comment]
Hey, I have exactly the same problem..

**Comment by AndyFicile** [comment]
> The server works on ubuntu but I never tested on debian so it probably doesn't work that well on it. This all might be fixed in the next update, but at the moment, I don't know why that is happening.

I've just tried it, I have the same problem on Ubuntu

**Comment by SineVector241** [comment]
Yeah. I don't know what's going on there, sorry. I've stopped maintaining the current version and am heavily working on the new version so I suggest waiting for that to come out since HttpListener in .NET isn't the best...

**Comment by AndyFicile** [comment]
No problem, do you know when you will release the new version ?

**Comment by SineVector241** [comment]
Not for like 2 or 3 months. You can check the progress on the dev branch. Here's a picture of it.![Screenshot_2024-12-11-12-26-24-99_9d48979aec2a60fefa8697c6986f545e.jpg](https://github.com/user-attachments/assets/23fbe7a6-2bbf-4922-8112-96b9f5f6561d)

**Comment by AndyFicile** [comment]
Okay, I will use the 1.0.6 version in the meantime.

**Comment by Miniontoby** [comment]
Have you guys tried running with `sudo`?

**Comment by Miniontoby** [comment]
I have a raspberry pi 4, I believe that also was Debian, right?

So I could try it out (if I would be able to download and install dotnet)...

I don't think I will be able to fix anything tho, but we'll see.


Perhaps it is an idea for you guys to run with docker. I have tested Docker before and am actively using it for VoiceCraft.
But I will still try it on my raspberry pi and then come back if I have more info
